### PR TITLE
Set up JS agent error Rollbar monitoring

### PR DESCRIPTION
**Pull request by Finesse**
Resolves https://fpjs.myjetbrains.com/youtrack/issue/JSAGENT-145

JS agent errors require a separate not-throttle monitoring channel. I and Artem have decided to use the [existing website Rollbar monitoring system](https://rollbar.com/fpjs/all/items/?rojects=319827) and add an explicit trait to JS agent errors to make them easier to find.

### CODE REVIEW

File: src/components/FpjsWidget/index.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default memo(function FpjsWidget() {
           setCurrentVisit(visits[0])
         }
       } catch (e) {
-        rollbar.error('Unable to initialize FingerprintJS Pro', e)
```
Comment (Finesse) [inline src/components/FpjsWidget/index.tsx:46]: Just a minor mistake that I've noticed

**Review by Lisenish — APPROVED** [review]

**Comment by Finesse** [comment]
@Lisenish Will you merge it? Or am I supposed to merge it?

**Comment by Lisenish** [comment]
> @Lisenish Will you merge it? Or am I supposed to merge it?

Merged but you could do that by yourself yeah. I actually wanted to do it yesterday after I approved it, but the branch was outdated, I synced it but then I forgot about it because I got distracted by other issues, sorry

**Comment by Finesse** [comment]
@Lisenish No problem, I want to clarify just in case

**Pull request merged**